### PR TITLE
Use Remote Build Execution for Bazel builds.

### PR DESCRIPTION
**Pull request by neuromage**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1031)

### CODE REVIEW

**Comment by neuromage** [comment]
This is now ready to be looked at.

/assign @IronPan

**Comment by neuromage** [comment]
@IronPan friendly ping

**Comment by neuromage** [comment]
@IronPan can you take a look at this one please? Thanks.

File: tools/bazel_builder/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+approvers:
+  - neuromage
+  - IronPan
+  - vicaire
```
Comment (IronPan) [inline tools/bazel_builder/OWNERS]: could you update this file?

Comment (neuromage) [reply tools/bazel_builder/OWNERS]: Done.

**Comment by Ark-kun** [comment]
@neuromage  Shall we merge this?

**Comment by neuromage** [comment]
Thanks @Ark-kun! Yes, we should merge it.

**Comment by Ark-kun** [comment]
> Thanks @Ark-kun! Yes, we should merge it.

AFAIK, You've put this PR on hold some time ago.

**Pull request merged**